src/dex-helper/icache.ts

```diff
@@ -12,4 +12,19 @@ export interface ICache {
     seconds: number,
     value: string,
   ): Promise<void>;
+
+  getAndCacheLocally(
+    dexKey: string,
+    network: number,
+    cacheKey: string,
+    ttl: number,
```

Comment:
I'd call it `seconds` for consistency with `setex`.

Reply:
Or change the others to `ttl` that works too...

Reply:
I thought I renamed ttl everywhere ^^'